core/gpslocation.cpp

```diff
@@ -39,7 +31,7 @@ GpsLocation::GpsLocation(void (*showMsgCB)(const char *), QObject *parent) :
 	loadFromStorage();
 
 	// register changes in time threshold
-	connect(qPrefLocationService::instance(), SIGNAL(qPrefLocationService::time_thresholdChanged()), this, SLOT(setGpsTimeThreshold(int seconds)));
+	connect(qPrefLocationService::instance(), SIGNAL(time_thresholdChanged(int)), this, SLOT(setGpsTimeThreshold(int)));
```

Comment:
You could use the pointer-to-member function version, to get at lease *some* compile-time verification:
```
connect(qPrefLocationService::instance(), &qPrefLocationService::time_thresholdChanged, this, &GpsLocation::setGpsTimeThreshold);
```

Apart from that minor thing - lgtm.

Reply:
Yes, I generally like the new syntax better. I had some problems with it a couple of times when trying to connect to overloaded slots and just recently learned that there's a special, magic helper for that 🙄 
But it's fine to not update this as part of this PR